hicommonwealth/commonwealth
# Remove Chain-Events Substrate Core

## Link to Issue
Closes: #3991 

## Description of Changes
- Removes all Substrate code used to subscribe and process Substrate chain blocks, events, and entities. This PR leaves only labeling functionality so that existing links to Substrate chain proposals and proposal loading works.

## Test Plan
- Deploy to chain-events-staging
- Deploy to Frick
- Ensure proposals on Substrate communities like `edgeware`, `chainx`, and dydx still load properly.
- Check that EVM and Cosmos listeners are running as usual by either viewing the CE-staging subscriber logs or by checking the database for new events after some time.

## Deployment Plan
Normal deployment.

## Other Considerations
- Email `timothee@common.xyz` with questions.
